TCAT/Utilities/SearchTableViewHelpers.swift

```diff
@@ -58,15 +58,18 @@ class SearchTableViewManager {
 
     private init() {}
     func getAllStops() -> [BusStop] {
-        if let stops = allStops {
-            return stops
+        if let stops = allStops{
```

Comment:
space after `allStops {` Make sure to run `swiftlint autocorrect` before finishing this PR